src/pages/Lend/LoanSearchPage.vue

```diff
@@ -13,7 +13,7 @@
 							</p>
 						</div>
 						<button class="tw-mb-2 tw-mt-3 tw-border-r tw-border-tertiary tw-px-1 md:tw-px-2">
-							<kv-material-icon :icon="mdiEarth" class="tw-text-secondary tw-w-3 tw-h-3" />
+							<kv-material-icon :icon="mdiEarth" class="tw-text-brand tw-w-3 tw-h-3" />
```

Comment:
It looks like this change is making both icons green/selected, is that what we want?